sqlserver/src/main/scala/zio/sql/sqlserver/SqlServerModule.scala

```diff
@@ -8,47 +8,49 @@ trait SqlServerModule extends Jdbc { self =>
     val builder = new StringBuilder
 
     def buildExpr[A, B](expr: self.Expr[_, A, B]): Unit = expr match {
-      case Expr.Source(tableName, column)                               =>
+      case Expr.Source(tableName, column)                                      =>
         val _ = builder.append(tableName).append(".").append(column.name)
-      case Expr.Unary(base, op)                                         =>
+      case Expr.Unary(base, op)                                                =>
         val _ = builder.append(" ").append(op.symbol)
         buildExpr(base)
-      case Expr.Property(base, op)                                      =>
+      case Expr.Property(base, op)                                             =>
         buildExpr(base)
         val _ = builder.append(" ").append(op.symbol)
-      case Expr.Binary(left, right, op)                                 =>
+      case Expr.Binary(left, right, op)                                        =>
         buildExpr(left)
         builder.append(" ").append(op.symbol).append(" ")
         buildExpr(right)
-      case Expr.Relational(left, right, op)                             =>
+      case Expr.Relational(left, right, op)                                    =>
         buildExpr(left)
         builder.append(" ").append(op.symbol).append(" ")
         buildExpr(right)
-      case Expr.In(value, set)                                          =>
+      case Expr.In(value, set)                                                 =>
         buildExpr(value)
         buildReadString(set)
-      case Expr.Literal(value)                                          =>
+      case Expr.Literal(value)                                                 =>
         val _ = builder.append(value.toString) //todo fix escaping
-      case Expr.AggregationCall(param, aggregation)                     =>
+      case Expr.AggregationCall(param, aggregation)                            =>
         builder.append(aggregation.name.name)
         builder.append("(")
         buildExpr(param)
         val _ = builder.append(")")
-      case Expr.FunctionCall0(function)                                 =>
+      case Expr.FunctionCall0(function) if (function.name.name == "localtime") =>
```

Comment:
We don't need this, as `localtime` exists only for PostgreSQL (at least for now)